.travis.yml

```diff
@@ -31,27 +31,3 @@ jobs:
           secure: "cVKcMHHv6MV1HiP78hYPyBx6VIy+DzIv6HS0QM/g3gHnE7EWopu9mQjku2DQtQWAjqVrIPL3ClvZxJH5MjDd4/kB6kWQDGX5O2+CaqzhaJgSpT0htFLbiClqK1uvl2QnHoqwfnhIux230AWDtQfod1g1p60nryo65pLG0HGHqXU="
         on:
           branch: master
-
-    - stage: Deploy to GH Releases
-
-      php: 7.2
-
-      before_install:
-        - source .travis/xdebug.sh
-        - xdebug-disable
-
-      script: skip
-
-      before_deploy:
-        - ./.travis/before-deploy-gh-release.sh
```

Comment:
That is, unless this script would be slowed down by enabling Xdebug.

Reply:
Probably slows it down - but I would imagine not worryingly